incubator/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -4193,7 +4176,7 @@ private void onSubscribeWindow(
                             iterator.remove();
                         }
 
-                        sendNewSessionStateForUnsubscribe(traceId, authorization, ackedTopicFilters);
+                        sendNewSessionStateForUnsubscribe(traceId, ackedTopicFilters);
```

Comment:
Does this method name need to be so specific, should it just be `doSendSessionState`?